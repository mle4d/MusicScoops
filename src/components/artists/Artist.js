import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

function Artist({ 
  Id,
  artist 
}) {
  return (
    <>
      <Link to={`/releases/${artist}/${Id}`}><h3 >{artist}</h3></Link> 
    </>
  );
}

Artist.propTypes = {
  Id: PropTypes.string.isRequired,
  artist: PropTypes.string.isRequired,
};

export default Artist;
