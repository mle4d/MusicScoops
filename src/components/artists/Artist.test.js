import React from 'react';
import { shallow } from 'enzyme';
import Artist from './Artist';

describe('Artist component', () => {
  it('renders App', () => {
    const wrapper = shallow(<Artist />);
    expect(wrapper).toMatchSnapshot();
  });
});
