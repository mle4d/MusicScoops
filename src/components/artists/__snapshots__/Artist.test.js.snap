// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Artist component renders App 1`] = `
<Fragment>
  <Link
    to="/releases/undefined/undefined"
  >
    <h3 />
  </Link>
</Fragment>
`;
